subsys/net/l2/ieee802154/Kconfig.radio

```diff
@@ -45,24 +45,25 @@ if NET_L2_IEEE802154_RADIO_CSMA_CA
 config NET_L2_IEEE802154_RADIO_CSMA_CA_MAX_BO
 	int "CSMA maximum backoffs"
 	default 4
-	range 1 5
+	range 0 5
 	help
 	  The maximum number of backoffs the CSMA-CA algorithm will attempt
 	  before declaring a channel access failure (MAC PIB attribute:
-	  maxMaxCSMABackoffs).
+	  macMaxCSMABackoffs).
 
 config NET_L2_IEEE802154_RADIO_CSMA_CA_MIN_BE
 	int "CSMA MAC minimum backoff exponent"
 	default 3
-	range 1 8
+	range 0 8
 	help
 	  The minimum value of the backoff exponent (BE) in the CSMA-CA
-	  algorithm (MAC PIB attribute: macMinBe).
+	  algorithm (MAC PIB attribute: macMinBe). This setting must be less
+	  than or equal NET_L2_IEEE802154_RADIO_CSMA_CA_MAX_BO.
```

Comment:
Did you mean `NET_L2_IEEE802154_RADIO_CSMA_CA_MAX_BE` here?

Reply:
Good catch. Fixed.